python3/kinesis/src/lambda_function.py

```diff
@@ -38,7 +38,7 @@ def _get_type(data):
         json_data = json.loads(data)
         return json_data["source"].split('.')[1]
     except (KeyError, ValueError):
-        return "kinesis_lambda"
+        return "logzio_kinesis_stream"
```

Comment:
It's better to keep it `kinesis_lambda` for backward compatibility, even if it's not aligned with CW.

Reply:
Opened a PR in the docs about it @imnotashrimp